shell/platform/darwin/ios/ios_external_texture_metal.mm

```diff
@@ -122,11 +239,6 @@ GrBackendTexture skia_backend_texture(texture_size.width(),   // width
                                         captures.release()         // release context
 
   );
-
-  if (!image) {
-    FML_DLOG(ERROR) << "Could not wrap Metal texture as a Skia image.";
-  }
```

Comment:
Why are you removing this?

Reply:
This line of code is still on line 79, it should be github diff to make it look like it has been deleted